### PR TITLE
Added _nav_tabs to Tags#comment Fixes #10570

### LINKED ISSUE #10570: Display topic comments below a tag page header with tabs
@TildaDares completed #10486 in #10559, and it looks great. 

The only thing left is that this doesn't display the comments page under the standard tag page header as shown above. Once you click "Comments" we just display a page of comments. I'd like to try rendering the comments below a tag page header as above, and will open a new issue for that!

_Originally posted by @jywarren in https://github.com/publiclab/plots2/issues/10486#issuecomment-993654695_

### PR DESCRIPTION
**Pull request by billymoroney1**
Fixes #0000 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -66,4 +66,10 @@ yarn-error.log
...
 .yarn-integrity
-.rake_tasks~
\ No newline at end of file
+.rake_tasks~
+/public/packs
+/public/packs-test
+/node_modules
+/yarn-error.log
+yarn-debug.log*
+.yarn-integrity
```
Comment (TildaDares) [inline .gitignore:75]: Hi @billymoroney1, can you remove this file, `application.js`, `development.rb`, `yarn.lock` and `package.json`? Those files are not needed in this PR. Thanks!

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (TildaDares) [reply .gitignore:75]: Also, can you add the issue number like this `Fixes #0000`? Thanks!

Comment (billymoroney1) [reply .gitignore:75]: or should i delete those in the PR itself? (below)

Comment (billymoroney1) [reply .gitignore:75]: @TildaDares Done (i deleted those files in the PR, if that's what you meant)

Comment (jywarren) [reply .gitignore:75]: Hi @billymoroney1 i think you may need to re-add them based on the main publiclab branch. Here it looks like your PR proposes deleting them from the project. If you just add the exact files as they are in the main branch, it should detect that they are back in sync and no longer appear in the PR. 

Also, I think it's best that you don't include these .gitignore additions, as they're not part of the feature you're working on (our general principle is one "project" per PR). 

Thank you! And no worries, a first PR is a lot of learning curve, it'll go smoothly more and more as you go! Thanks for your time!!! 🎉

Comment (billymoroney1) [reply .gitignore:75]: @jywarren AH i found the public lab github guidelines and forgot to merge my branch back with my fork of the repo before pushing. I'll go try that -

Comment (billymoroney1) [reply .gitignore:75]: (it's such a mess... i might try to backtrack and start clean and do it right)

**Comment by jywarren** [comment]
Hi @billymoroney1 did you open a new PR to start fresh? Or did you need some help? Thanks!

**Comment by billymoroney1** [comment]
@jywarren im gonna get the new PR going today! disappeared for xmas for a few days

**Comment by jywarren** [comment]
Oh no rush at all, really don't feel you need to get back into this now during holidays. Just wanted to be sure you weren't stuck or what. I can help out next week too!

**Pull request closed without merging**